.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,99 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+#   schedule:
+#     - cron: '21 13 * * *'
+  push:
```

Comment:
the current tag releases are `X.Y.Z` without the leading `v`. Would it work without `v` too? We could also do it not by tag but for release, as done in the other workflows